SR2019/2019-01-28-feb-techday.md

```diff
@@ -0,0 +1,21 @@
+---
+to: Student Robotics 2019
+subject: February tech day reminder
+---
+
+Hi all,
+
+Just a reminder and some information on the tech day we're running on on Saturday 9th February.
+
+Tech Days are opportunities for you to get help if you're struggling with a technical issue, and is a controlled environment to make more progress on your robots. It’s also an opportunity to see how well you're doing in comparison to other teams.
+
+We're running this tech day in 2 locations, London and Southampton. The day will run from 10am until 5pm, however feel free to come and go as you please.
+
+The London venue will only run of there is sufficient interest. Spaces are limited to six teams, so be sure to let us know if you’d like to attend.
```

Comment:
Since we already have two teams who would be interested in coming to London, perhaps drop the first sentence here? Just a thought.

Reply:
Quite possibly, but it means teams remember this for future events. It also means the next reminder email I can just copy most of the content without remembering to add this back!